config/prod/recover-s3-bucket.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.8/templates/S3/synapse-external-bucket.j2"
+stack_name: "recover-s3-bucket"
+stack_tags:
+  Department: "SysBio"
+  Project: "recover"
+  OwnerEmail: 'meghasyam@sagebase.org'
+  CostCenter: "MGH RECOVER / 122500"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
```

Comment:
forgot one more thing here.  if you need access to this bucket you'll want to add an ARN for yourself.  so adding this ARN will do it.. `arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/meghasyam@sagebase.org`

Reply:
Thanks Khai! Will add this too